test/e2e/app-dir/css-order/app/vendor/a/page.tsx

```diff
@@ -0,0 +1,13 @@
+import { SideEffectsArrayComponent } from 'side-effects-array-dep'
+import Nav from '../../nav'
+
+export default function Page() {
+  return (
+    <div>
+      <SideEffectsArrayComponent id="vendor-side-effects-array">
```

Comment:
It probably also has some influence whether these components are client components or server components. We might want to double the test case for client components too.

Reply:
working on that!